foreman_remote_execution.gemspec

```diff
@@ -26,7 +26,6 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'deface'
   s.add_dependency 'dynflow', '>= 1.0.2', '< 2.0.0'
-  s.add_dependency 'foreman-tasks', '>= 8.3.0'
```

Comment:
Technically I think you could even keep this, as long as the `gem` line in `gemfile.d` also satisfies it.